pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -400,7 +400,7 @@ func (s *Service) ReconcileLoadBalancerMember(openStackCluster *infrav1.OpenStac
 	}
 
 	loadBalancerName := getLoadBalancerName(clusterName)
-	s.scope.Logger().Info("Reconciling load balancer member", "loadBalancerName", loadBalancerName)
+	s.scope.Logger().Info("Reconciling load balancer member", "name", loadBalancerName)
```

Comment:
Tricky. The only parameter doesn't refer to the subject of the log message. i.e. We're creating a 'member', but `name` refers to the load balancer, not the member. I'd probably use `loadBalancerName` here.